tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```diff
@@ -606,9 +608,9 @@ class ReplicationUtility {
                     `Unexpected CRR failure occurred: ${JSON.stringify(data)}`,
                 );
                 if (status === 'PENDING' || status === 'PROCESSING') {
-                    return setTimeout(callback, 2000);
+                    return setTimeout(cbOnce, 2000);
                 }
-                return callback();
+                return cbOnce();
```

Comment:
should we take the chance to cleanup the code: and pass the `status` from this callback to the next (instead of using a local variable?

```suggestion
                if (status === 'PENDING' || status === 'PROCESSING') {
                    return setTimeout(() => cbOnce(nil, status), 2000);
                }
                return cbOnce(nil, status);
            }),
            () => (status === 'PENDING' || status === 'PROCESSING'),
```